### PR TITLE
Support of 'scw {images,ps} --filter' (#134)

### PR DESCRIPTION
**Pull request by moul**

### CODE REVIEW

**Comment by QuentinPerez** [comment]
LGTM :+1:

File: pkg/cli/cmd_images.go [hunk 1]
```diff
@@ -38,6 +63,17 @@ func runImages(cmd *Command, rawArgs []string) error {
 		All:     imagesA,
 		Quiet:   imagesQ,
 		NoTrunc: imagesNoTrunc,
+		Filters: make(map[string]string, 0),
+	}
+	if imagesFilters != "" {
+		for _, filter := range strings.Split(imagesFilters, " ") {
+			parts := strings.SplitN(filter, "=", 2)
```
Comment (aimxhaisse) [inline pkg/cli/cmd_images.go]: I'd add an extra check to ensure len(parts) == 2

Comment (moul) [inline pkg/cli/cmd_images.go]: good catch dude

Comment (moul) [inline pkg/cli/cmd_images.go]: fixed with 672c193

**Comment by aimxhaisse** [comment]
LGTM

**Pull request merged**